docs/topic_guides/backends.md

```diff
@@ -12,22 +12,60 @@ tags:
 
 Splink is a Python library. It implements all data linking computations by generating SQL, and submitting the SQL statements to a backend of the user's chosing for execution.
 
-For smaller input datasets of up to 1-2 million records, users can link data in Python on their laptop using the DuckDB backend. This is the recommended approach because the DuckDB backend is installed automatically when the user installs Splink using `pip install splink`. No additional configuration is needed.
-
-Linking larger datasets requires highly computationally intensive calculations, and generates datasets which are too large to be processed on a standard laptop. For these scenarios, we recommend using one of Splink's big data backend - currently Spark or AWS Athena. When these backends are used, the SQL generated by Splink is sent to the chosen backend for execution.
-
 The Splink code you write is almost identical between backends, so it's straightforward to migrate between backends. Often, it's a good idea to start working using DuckDB on a sample of data, because it will produce results very quickly. When you're comfortable with your model, you may wish to migrate to a big data backend to estimate/predict on the full dataset.
 
 ## Choosing a backend
 
+### Considerations when choosing a SQL backend for Splink
+When choosing which backend to use when getting started with Splink, there are another factors to consider:
+
+- the size of the datasets
+- the amount of configuration required
```

Comment:
This is ambiguous, but you may well address it below and I just haven't got there yet.